sakisandrac/overlook-final-project
# Fix/login loading

## Additions to project:
- Login page now works (without loading screen)
- Bookings are sorted according to past and current bookings
- Styling was added to dashboard and new bookings view
- User can filter by room type 
- User can book room after searching for a date, clicking a room and then clicking reserve now
- User can see total spent on room bookings

## How to test features:
- Log in with the username: customer50 password: overlook2021
- Go to new bookings page, and search for a booking date
- Click on a room type to filter by room type
- Click on book now to go to see details of the room
- Click on reserve now- you will see a confirmation number if room was booked successfully and if you go to the dashboard, the booking should now show up under current bookings and the total spent should be updated as well.

## Features that are not finished yet:
- There is no way to exit out of the "book now" or single room view, so to go back you'll have to click on the search dates button again.
